src/main.ts

```diff
@@ -79,10 +96,10 @@ async function main() {
     }
 }
 
-async function executeAzCliCommand(command: string, silent?: boolean, options: any = {}) {
+async function executeAzCliCommand(command: string, silent?: boolean, options: any = {}, parameters: any = []) {

```

Comment:
having both options and parameters might be confusing. parameters seems like Az cli arguments to me and options refer to execOptions. Can we update this ?

Reply:
"parameters" must be renamed to arguments?

Reply:
made changes